docker-compose.yml

```diff
@@ -21,6 +21,20 @@ services:
       MYSQL_ROOT_PASSWORD: password
     restart: unless-stopped
 
+  phpmyadmin:
```

Comment:
does PMA have an error log? if so, can we make sure the column is synced like the other containers?

Reply:
It doesn't! At least not that I could find out about anywhere!